crates/proof-of-sql/src/base/scalar/mont_scalar.rs

```diff
@@ -187,7 +190,14 @@ impl<T: MontConfig<4>> From<[u64; 4]> for MontScalar<T> {
         Self(Fp::new(ark_ff::BigInt(value)))
     }
 }
-
+impl<T: MontConfig<4>> From<MontScalar<T>> for BigInt {
+    fn from(val: MontScalar<T>) -> Self {
+        println!("val: {:?}", val.0);
+        let raw: [u8; 32] = bytemuck::cast(val.0 .0 .0);
```

Comment:
Use `Into<[u64; 4]>`.

Reply:
@JayWhite2357 Of course! That was taken care of in #78.